identity-enabler/deviceId-mobile-app/src/pages/Tutorial.svelte

```diff
@@ -6,45 +6,45 @@
     import { loadingScreen } from "../lib/store";
     import { showAlert } from "../lib/ui/helpers";
 
-    let code = "";
+    let markdown = "";
+    let title = "";
 
     onMount(async () => {
         loadingScreen.set("Loading...");
         try {
-            code = await getMarkdownContent(TUTORIAL_URL);
+            const tutorialMd: string = await getMarkdownContent(TUTORIAL_URL);
+            const endOfTitleIdx = tutorialMd.indexOf("\n");
+            title = tutorialMd.substring(2, endOfTitleIdx);
+            markdown = tutorialMd.substring(endOfTitleIdx + 1);
```

Comment:
is this really necessary, to depend on the structure of the input tutorial? can we just have a title and avoid potential code breaking if the tutorial structure changes?

Reply:
Yes, but then there are two titles, one in the header and one in the rendered markdown.